docker/zts/systemd/start.sh

```diff
@@ -3,7 +3,10 @@
 LIB_FILE="$(dirname $(readlink --canonicalize "$0"))/../../lib.sh"
 . $LIB_FILE
 
-DOCKER_CMD="docker run --publish 1969:1969"
+# We use syslog as log driver because it's the easiest way to redirect output to a single known file
+# which can later be made accessible via zotero_cgi.
+# (Adding apache user to docker group & calling `docker logs` is not suitable under CentOS 8).
```

Comment:
"the" docker group...
Do you mean "appropriate" by "suitable"?

Reply:
I mean that it is allowed, but it won't work anyway.
https://www.projectatomic.io/blog/2015/08/why-we-dont-let-non-root-users-run-docker-in-centos-fedora-or-rhel/